src/metamask/sign.ts

```diff
@@ -16,3 +16,17 @@ export const sign =
 
     await clickOnButton(page, "Sign");
   };
+
+export const signTypedData =
+  (page: DappeteerPage, getSingedIn: GetSingedIn) =>
+  async (): Promise<void> => {
+    await page.bringToFront();
+    if (!(await getSingedIn())) {
+      throw new Error("You haven't signed in yet");
+    }
+    await page.reload();
+    await page.waitForTimeout(300);
```

Comment:
Assume this is while metamask evaluates input data? Is there something else we can wait for as this could cause flakiness if metamask sometimes takes longer?

Reply:
totally, I did not spend too much time in trying to remove those `wait` for now, but I'll make sure it's taken care of.